modules/telco-upgrade-acknowledging-the-upgrade.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/day_2_core_cnf_clusters/upgrading/telco-upgrade-completing-the-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="telco-upgrade-acknowledging-upgrade-for-eus-eus-y-stream_{context}"]
+= Acknowledging the upgrade for EUS-EUS and y-stream upgrades
```

Comment:
I think "EUS-to-EUS" updates are now called "Control Plane Only"

https://83448--ocpdocs-pr.netlify.app/openshift-enterprise/latest/updating/updating_a_cluster/control-plane-only-update#control-plane-only-update